homeassistant/components/smartthings/media_player.py

```diff
@@ -0,0 +1,240 @@
+"""Support for media players through the SmartThings cloud API."""
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+from pysmartthings import Capability, DeviceEntity
+
+from homeassistant.components.media_player import (
+    MediaPlayerEntity,
+    MediaPlayerEntityFeature,
+    MediaPlayerState,
+    RepeatMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import STATE_PAUSED, STATE_PLAYING
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SmartThingsEntity
+from .const import DATA_BROKERS, DOMAIN
+
+CONTROLLABLE_SOURCES = ["bluetooth", "wifi"]
```

Comment:
add "bluetooth" and "wifi" to .const as variables to avoid magic values

Reply:
You probably can remove it as it is no longer used

Reply:
Sure, will do, thanks for the reminder!